src/test/ui/cmse-nonsecure-entry/params-on-stack.rs

```diff
@@ -0,0 +1,11 @@
+// gate-test-cmse_nonsecure_entry
```

Comment:
A feature gate test tests that an unstable feature is not usable without a `#![feature]` attribute, so you need to add a test that does so and move this comment there

```suggestion
```

Reply:
Added the gate-test and fixed the wrong header in other ones